dev/integration_tests/flutter_gallery/lib/demo/transformations/transformations_demo_gesture_transformable.dart

```diff
@@ -211,6 +211,8 @@ class _GestureTransformableState extends State<GestureTransformable> with Ticker
                     details.globalPosition - getOffset(context),
                     _transform,
                   ),
+                  kind: details.kind,
```

Comment:
NOTE: I am also wiring kind on places that I found it missing. Please let me know if the omission was intentional in these places.